src/assets/icon/PostToCommunity.js

```diff
@@ -37,4 +38,8 @@ const PostToCommunity = (props) => (
   </Svg>
 );
 
+PostToCommunity.propTypes = {
+  color: PropTypes.string
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding PropTypes validation for the `color` prop is a good practice. It helps ensure that the correct type of value is passed to the `color` prop, which can prevent potential bugs or crashes.
```diff
+ PostToCommunity.propTypes = {
+   color: PropTypes.string
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->